packages/adp-tooling/src/translations/adp-tooling.i18n.json

```diff
@@ -8,21 +8,35 @@
         "maxAgeTooltip": "(Optional) Enter the time in seconds it takes for a cached response of the service to expire.",
         "oDataAnnotationSourceURILabel": "OData Annotation Data Source URI",
         "oDataAnnotationSourceURITooltip": "Server side annotations have been detected for the selected OData service. If needed, enter new annotation data source URI  for the changed service",
+        "oDataURIInfoLabel": "Have in mind that common OData Service path prefixes are '{{- v2Prefix}}' or '{{- v4Prefix}}'",
+        "oDataURIV2InfoLabel": "Have in mind that you are trying to replace OData {{versionOne}} Service with OData {{versionTwo}} Service",
```

Comment:
I don't see oDataURIInfoLabel, oDataURIV2InfoLabel or inboundId used anywhere. Why do we have them here? Or did I miss a reference?

Reply:
Fixed in [23549bc](https://github.com/SAP/open-ux-tools/pull/2167/commits/23549bc071b644cff18414ef2f2b3f32fcdeb5bb)